src/PAModel/PAConvert/ErrorContainer.cs

```diff
@@ -51,18 +60,20 @@ IEnumerator IEnumerable.GetEnumerator()
         // Helper for writing out errors.
         public void Write(TextWriter output)
```

Comment:
Is the problem in the console output? The bug is related to the telemetry in PA-Client, which I had thought was logged from a different location (not from console). I don't think there's a need to limit the console output that I know of...

Reply:
This change has been made within `ErrorContainer` itself which is then passed onto wherever the errors need to be logged. It can be console, or in this case even Kusto (if I'm not mistaken) i.e., the error container (the source) is still common irrespective of where we're sending these logs.